draft-ietf-quic-transport.md

```diff
@@ -4051,38 +4052,42 @@ Network routing and path elements can however change mid-connection; an endpoint
 MUST disable ECN if validation later fails.
 
 
-# Packet Size {#packet-size}
+# Datagram Size {#datagram-size}
 
-The QUIC packet size includes the QUIC header and protected payload, but not the
-UDP or IP headers.
+A UDP datagram can include one or more QUIC packets. The datagram size refers to
+the total UDP payload size of a single UDP datagram carrying QUIC packets. The
+datagram size includes includes one or more QUIC packet headers and the
+protected payloads, but not the UDP or IP headers.
+
+The maximum datagram size is defined as the largest size of UDP payload that can
+be sent across a network path using a single UDP datagram.  The maximum datagram
+size MUST be at least 1200 bytes.
 
 QUIC depends upon a minimum IP packet size of at least 1280 bytes.  This is the
 IPv6 minimum size ({{?IPv6=RFC8200}}) and is also supported by most modern IPv4
-networks.  Assuming the minimum IP header size, this results in a QUIC maximum
-packet size of 1232 bytes for IPv6 and 1252 bytes for IPv4.
+networks.  Assuming the minimum IP header size, this results in a maximum
+datagram size of 1232 bytes for IPv6 and 1252 bytes for IPv4.
```

Comment:
Here it should say "IP packet" instead of datagram as it is the IP packet sizes for IPv4 and IPv6 that are given.

Reply:
Re-reading, I think this might have been better as QUIC maximum packet size, since that is accurate.

Reply:
It isn't the IP packet size or the QUIC packet size -- this really is about the UDP payload size (1280 - 20/40 - 8 = 1252/1232). I'm leaving this as is.

Reply:
I'm now spelling this out a bit more, so the computation is crystal clear.